content/scripts/build.js

```diff
@@ -1149,7 +1149,7 @@ class Builder {
             })
             .map(([uri, documentData]) => {
               if (!documentData.modified) {
-                throw new Error("No .modified in documentData");
+                throw new Error(`No .modified in documentData (url: ${uri})`);
```

Comment:
Unrelated but if it happens, without knowing the `uri` it's really hard to see what WHICH document had no `.modified`.